x/concentrated-liquidity/math_test.go

```diff
@@ -0,0 +1,22 @@
+package concentrated_liquidity_test
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	cl "github.com/osmosis-labs/osmosis/v12/x/concentrated-liquidity"
+)
+
+func (s *KeeperTestSuite) TestGetLiquidityFromAmounts() {
+	currentSqrtP, err := sdk.NewDecFromStr("70.710678")
+	sqrtPLow, err := sdk.NewDecFromStr("67.082039")
+	sqrtPHigh, err := sdk.NewDecFromStr("74.161984")
+	s.Require().NoError(err)
+
+	amount0Desired := sdk.NewInt(1)
+	amount1Desired := sdk.NewInt(5000)
+
+	s.SetupTest()
+
+	liquidity := cl.GetLiquidityFromAmounts(currentSqrtP, sqrtPLow, sqrtPHigh, amount0Desired, amount1Desired)
+	s.Require().Equal("1377.927096082029653542", liquidity.String())
```

Comment:
how do we get these numbers? Is this something that you would want to address later on / in future Prs?

Reply:
These numbers are retrieved by reverse engineering the CalcOutAmountGivenIn tests where we test for amount0 and amount1. I cross checked these values with what we were getting for liquidity values for that test